### PR TITLE
Add support for local registry

### PR DESCRIPTION
**Pull request by pravin-dsilva**
Signed-off-by: Pravin Dsilva <pravin.d-silva@ibm.com>

### CODE REVIEW

**Review by Prajyot-Parab — DISMISSED** [review]
/lgtm

File: modules/3_helpernode/templates/helpernode_vars.yaml [hunk 1]
```diff
@@ -54,3 +54,14 @@ ocp_installer: "${install_tarball}"
...
 ocp_install_kernel: "file:///dev/null"
+
+%{ if local_registry.enable_local_registry }
+setup_registry:
+  deploy: true
+  autosync_registry: true
+  registry_image: "${local_registry.registry_image}"
+  local_repo: "${local_registry.ocp_release_repo}"
+  product_repo: "openshift-release-dev"
+  release_name: "ocp-release"
```
Comment (yussufsh) [inline modules/3_helpernode/templates/helpernode_vars.yaml:65]: @Prajyot-Parab better have this as a variable? The way we had done this for PowerVS?

Comment (Prajyot-Parab) [reply modules/3_helpernode/templates/helpernode_vars.yaml:65]: yes, that makes sense but then in `ocp4-playbook`, we stick to this `quay.io/openshift-release-dev/ocp-release` so won't really matter. If @pravin-dsilva Can try to parameterize it in `ocp4-playbook` as well then we can make it a variable.

Comment (yussufsh) [reply modules/3_helpernode/templates/helpernode_vars.yaml:65]: Ok then it makes sense.. but you think it is good to make it a user variables please open new PRs.

Comment (Prajyot-Parab) [reply modules/3_helpernode/templates/helpernode_vars.yaml:65]: Sure we can get this PR merged and as a next step, I will focus on converting whatever remains into variables inline with ocp4-playbook changes.

**Review by yussufsh — APPROVED** [review]
lgtm

**Review by Prajyot-Parab — APPROVED** [review]

**Pull request merged**